index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="initial-scale=1, width=device-width" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" />
+  </head>
+  <body>
+    <header class="header">
+      <img class="logo" src="./logo.png" alt="logo">
+        <ul class="navbar">
+          <li class="nav-icon">Home</li>
+          <li class="nav-icon">My Projects</li>
+          <li class="nav-icon">Contact US</li>
+        </ul>
+    </header>
+    
+    <section class="main container">
+      <img class="mobile-icon" src="./public/headershapes-mobile.svg" />
+      <div class="headline">
+        <b class="primary-text">
+          <span>
+            <p class="im-qaisar">I’m Qaisar.</p>
+            <p class="im-qaisar">Glad to see you!</p>
```

Comment:
- [ ] I recommend using a `<h1>` tag for your main project title. This should only be used once for any project's main title. It'll also help in making your project become more semantic.